### PR TITLE
Fix #112: update Safari to Safari 11

### LINKED ISSUE #112: Test Safari Technology Preview
I believe the dashboard is testing Safari 10.0. A lot of WPT improvements went into Safari 10.1 and it is very unfortunate those are not reflected on the dashboard.

### PR DESCRIPTION
**Pull request by gsnedders**
Fixes #112.

### CODE REVIEW

**Comment by foolip** [comment]
Ah, so Sauce has Safari 11 now? @jeffcarp, can you review?

File: browsers.json [hunk 1]
```diff
@@ -56,10 +56,19 @@
         "os_version": "10",
         "sauce": true
     },
-    "safari-10-macos-10.12-sauce": {
+    "safari-11-macos-10.12-sauce": {
```
Comment (jeffcarp) [inline browsers.json]: I think to futureproof this can we call it `safari-11.0-macos-10.12-sauce`?

File: browsers.json [hunk 2]
```diff
@@ -56,10 +56,19 @@
...
         "initially_loaded": true,
         "currently_run": true,
         "browser_name": "safari",
+        "browser_version": "11",
```
Comment (jeffcarp) [inline browsers.json]: `11.0` here too

**Comment by foolip** [comment]
@mattl FYI, looks like we'll get Safari 11 with Sauce, but continuing to investigate BrowserStack is still the right path IMHO.

**Comment by gsnedders** [comment]
@jeffcarp done!

**Comment by jeffcarp** [comment]
Thanks! 👍

**Pull request merged**